src/config/transform.rs

```diff
@@ -273,9 +273,14 @@ pub fn get_transform_output_ids<T: TransformConfig + ?Sized>(
 ) -> impl Iterator<Item = OutputId> + '_ {
     transform
         .outputs(
-            vector_lib::enrichment::TableRegistry::default(),
+            &TransformContext {
+                schema: SchemaOptions {
+                    log_namespace: Some(global_log_namespace == LogNamespace::Vector),
```

Comment:
nit

Maybe this can be replaced by an `into()`. I had to dig to find out if this was correct or not.

`impl From<bool> for LogNamespace {` already exists and maybe `impl From<LogNamespace> for bool {` should too